src/scss/components/navigation/_index.scss

```diff
@@ -56,17 +59,28 @@
     text-decoration: none;
 
     &.current {
-      background-color: $gray-900;
-      color: $gray-0;
+      background-color: $gray-200;
+      border-left: 0.125rem solid $purple-500;
+      color: $purple-500;
+      margin-left: 1.25rem;
+      padding-left: 1.125rem;
     }
 
     &:hover {
-      background-color: $gray-800;
-      color: $gray-0;
+      background-color: $gray-200;
+      color: $purple-500;
     }
   }
 
   hx-reveal {
+    a.active {
+      background-color: $gray-200;
+      border-left: 0.125rem solid $purple-500;
+      color: $purple-500;
+      margin-left: 0.3rem;
+      padding-left: 3.25rem !important;
```

Comment:
We should avoid using `!important`...is it needed here?

Reply:
Yes I removed , Please review